test-cases/gutenberg/group.md

```diff
@@ -0,0 +1,228 @@
+
+--------------------------------------------------------------------------------
+
+##### TC003
+
+### Deep nesting is possible (iOS only)
+
+-   Add a `Group` block
+-   Nest blocks inside multiple `Group` blocks (at least 3 levels deep)
+-   Check if app do not crashes
+
+--------------------------------------------------------------------------------
+
+##### TC004
+
+### Check if Group placeholder is render on RootList
+
+-   Add a `Group` block
+-   Deselect
+-   Check if you are able to see `Group` placeholder
+
+Expected look:  
+![Progress](../resources/group-placeholder-root.png)
+
+--------------------------------------------------------------------------------
+
+##### TC005
+
+### Check if Group placeholder is render in nested structure
+
+-   Add a `Group` block
+-   Nest some `Group` inside
+-   Select top-most `Group` block in hierarchy
+-   Check if you are able to see `Group` placeholder of nested blocks wrapped with dashed border
+-   Go down in the hierarchy observing if margins is keept (the whole UI should remain the same)
+-   After selecting last `Group` you should be able to see it's `AppenderButton`
+
+Expected look:  
+![PlaceholderNestes](../resources/group-placeholder-nest-1.png)
+ - Post title is not dimmed
+ - Navigation arrows is not visible
+ - Only selected block title and icon is visible in breadcrumbs
+
+![PlaceholderDeepNested](../resources/group-placeholder-nest-deep.png)
+ - Post title is dimmed
+ - Navigation arrows is visible
+ - Navigation arrows are disable depending on order in block
+ - Show parent in breadcrumbs
+
+--------------------------------------------------------------------------------
+
+##### TC006
+
+### Nested block have proper border styling
+
+-   Add a `Group` block
+-   Nest some blocks inside
+-   Check if each selected block gets solid blue border
+-   Check if each first child gets dashed grey border
+
+Expected look:  
+![BorderStyling](../resources/group-border-styling.png)
+
+--------------------------------------------------------------------------------
+
+##### TC007
+
+### Nested block have proper margins values
+
+-   Add a `Group` block
+-   Nest some blocks inside
+-   Check if margins and paddings do not bounce when changing selection (exception is the trasition of the selected block connected with making space for FloatingToolbar)
+-   Check if there is no weird/unexpected extra free space around blocks
+-   Check the margins/paddings values
+
+Expected look:  
+![MarginRoot](../resources/group-margin-root.png)
+ - Margins of `3px` left/right (between screen edge and solid border)
+
+![MarginGroupChild](../resources/group-margin-inner-child-group.png)
+ - Margins of `5px` left/right (beetwen dashed border and solid border)
+
+![MarginChild](../resources/group-margin-child.png)
+ - Margin of `6px` left/right (beetwen content and dashed border)
+
+![MarginContent](../resources/group-margin-root-1.png)
+ - Margin of `16px` left/right
```

Comment:
Do we really mean "margin" here? Since "margin" is a technical term let's use it only when we refer to a real margin. otherwise we can use "space"

Reply:
ok I have change wording